canonical-web-and-design/charmhub.io
# Add new footer

## Done

- Replaced the existing footer with one identical to juju.is

## How to QA
- Visit the demo in your browser, link commented below
  - Or you can run the project locally using the [dotrun](https://snapcraft.io/dotrun) snap with `$ dotrun` and view it in your web browser at: http://localhost:8045/
  - If your demo doesn't work, read more about the [demoservice](https://discourse.ubuntu.com/t/demoservice/16862)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the footer matches [the design](https://app.zeplin.io/project/606203889af57e12287b5308/screen/606edfa32c8c0b3ac8935e2c)

## Issue / Card
Fixes https://github.com/canonical-web-and-design/snap-squad/issues/2005

## Screenshots
![Screenshot from 2021-04-23 12-31-36](https://user-images.githubusercontent.com/2376968/115865174-e1952c00-a42f-11eb-9750-8ab3f8f0c4fa.png)
